app/src/main/res/values/strings.xml

```diff
@@ -19,6 +19,7 @@
     <string name="pin_conversation">Pin to the top</string>
     <string name="unpin_conversation">Unpin</string>
     <string name="forward_message">Forward</string>
+    <string name="compress_error">Unable to compress image to selected size</string>
```

Comment:
if you add a new string, add it into all language files please

Reply:
> Is there a tool to automate this process ? It's tedious to copy/paste in each file)

Reply:
I have a linux script for it, no other helper

Reply:
Done :)

Reply:
(I just mixed one of my accounts)